app/src/pages/profilePage/personalInfoBlock/personalInfoBlock.jsx

```diff
@@ -0,0 +1,169 @@
+    const height = image.height;
+    const size = Math.floor(file.size / 1024);
+    return size <= 1000 && width <= 300 && height <= 500;
+  };
+
+
+  render() {
+    // const xhr = new XMLHttpRequest();
+    // const url = `/api/v1/data/photo?superadmin?at=${Date.now()}` +
+    //  `&access_token=14113ab4-9c8f-421f-a203-fe86b7c964f6`;
+    // xhr.open('GET', url, false);
+    // xhr.send();
+    // xhr.onreadystatechange = function () { // (3)
+    //   if (xhr.readyState !== 4) return;
+    //   if (xhr.status === 200) {
+    //     console.log(xhr);
+    //   }
+    // };
+
+    return (
+      <div className={cx('personal-info-block')}>
+        <BlockContainer.Header>
+          <FormattedMessage id={'PersonalInfoBlock.header'} defaultMessage={'Personal information'} />
+        </BlockContainer.Header>
+        <BlockContainer.Body>
+          <div className={cx('block-content')}>
+            <div className={cx('avatar-wrapper')}>
+              <img className={cx('avatar')} src={this.state.avatarSource} alt="Profile avatar" />
+            </div>
+            <div className={cx('info')}>
+              <div className={cx('login')}>
+                { this.props.login }
+              </div>
+              <div className={cx('name-field')}>
+                <span className={cx('name')}>
+                  { this.props.name }
+                  <i className={cx('name-pencil-icon')} dangerouslySetInnerHTML={{ __html: PencilIcon }} />
+                </span>
+              </div>
+              <div className={cx('email')}>
+                { this.props.email }
+                { <i className={cx('email-pencil-icon')} dangerouslySetInnerHTML={{ __html: PencilIcon }} /> }
+              </div>
+              <div className={cx('photo-controls')}>
+                <div className={cx('input-file')}>
+                  <input ref={(inputFile) => { this.fileSelector = inputFile; }} onChange={this.selectPhotoHandler} type="file" accept="image/gif, image/jpeg, image/png" />
+                </div>
+                <div className={cx('photo-btn')}>
+                  <GhostButton onClick={this.onClickUploadPhoto}>
+                    <FormattedMessage id={'PersonalInfoBlock.uploadPhoto'} defaultMessage={'Upload photo'} />
+                  </GhostButton>
+                </div>
+                <div className={cx('photo-btn')}>
+                  <GhostButton>
+                    <FormattedMessage id={'PersonalInfoBlock.removePhoto'} defaultMessage={'Remove photo'} />
+                  </GhostButton>
+                </div>
+              </div>
+            </div>
+
+            <div className={cx('change-pass-btn')}>
+              <GhostButton>
+                <FormattedMessage id={'PersonalInfoBlock.changePassword'} defaultMessage={'Change password'} />
+              </GhostButton>
+            </div>
+
+          </div>
+        </BlockContainer.Body>
+      </div>
+    );
+  }
+
+}
+
+PersonalInfoBlock.propTypes = {
```

Comment:
For components written as a class we can define theit prop types and default props as a class static fields:
```
class MyComponent extends Component {
  static propTypes = {
    ...
  };

  static defaultProps = {
    ...
  };

  ...
}
```

So we can see component's interface near it's definition.